### PR TITLE
Added multi-value methods to request util

### PR DESCRIPTION
**Pull request by sschmittBt**

### CODE REVIEW

File: src/main/java/de/braintags/vertx/util/request/RequestUtil.java [hunk 1]
```diff
@@ -207,6 +239,20 @@ public static String readFormAttribute(RoutingContext context, String key, Strin
...
+   * Read all values of the defined key from a transferred form request
+   *
+   * @param context
+   *          the context to be handled
+   * @param key
+   *          the key to retrive from the form parameters
+   * @return the values
+   */
+  public static List<String> readFormAttributes(final RoutingContext context, final String key) {
+    List<String> values = context.request().formAttributes().getAll(key);
```
Comment (remi128) [inline src/main/java/de/braintags/vertx/util/request/RequestUtil.java]: better direct return

File: src/main/java/de/braintags/vertx/util/request/RequestUtil.java [hunk 2]
```diff
@@ -117,6 +118,20 @@ public static String readParameterAttribute(RoutingContext context, String key,
...
+   * Read all values of the defined key from a request parameter
+   *
+   * @param context
+   *          the context to be handled
+   * @param key
+   *          the key to retrive from the request parameters
+   * @return the values
+   */
+  public static List<String> readParameterAttributes(final RoutingContext context, final String key) {
+    List<String> values = context.request().params().getAll(key);
```
Comment (remi128) [inline src/main/java/de/braintags/vertx/util/request/RequestUtil.java]: better direct return

**Review by remi128 — CHANGES_REQUESTED** [review]

File: src/main/java/de/braintags/vertx/util/request/RequestUtil.java (hunk 2 above)
Comment (sschmittBt) [reply src/main/java/de/braintags/vertx/util/request/RequestUtil.java]: fixed

File: src/main/java/de/braintags/vertx/util/request/RequestUtil.java (hunk 1 above)
Comment (sschmittBt) [reply src/main/java/de/braintags/vertx/util/request/RequestUtil.java]: fixed

**Review by remi128 — APPROVED** [review]

**Pull request merged**